src/lib/util/ArgumentParser.cpp

```diff
@@ -32,6 +32,13 @@ void ArgumentParser::addParameter(const String &name, const String &abbreviation
     }
 }
 
+/* Displays the whole command entered by the user in the shell */
+String ArgumentParser::getEnteredCommand(){
+    String command = "";
+    for(auto it:unnamedArguments) command += it + " ";
```

Comment:
We think that single line loops are not very readable and would prefer use of brackets and blocks.